README.md

```diff
@@ -61,42 +72,69 @@ cd aws-iot-device-sdk-java-v2
 mvn clean install
 ```
 
-If you wish to use the latest CRT release, rather than the latest tested with the IoT SDK, you can run the following before running `mvn clean install`:
+If you wish to use the latest CRT release, rather than the latest tested with the IoT SDK, you can run the following command before running `mvn clean install`:
 
-~~~ sh
+``` sh
 # Update the version of the CRT being used
 mvn versions:use-latest-versions -Dincludes="software.amazon.awssdk.crt*"
-~~~
+```
+
+#### Building AWS CRT from source
+
+If you also need to build AWS CRT Java from source, visit [AWS CRT Java](https://github.com/awslabs/aws-crt-java?tab=readme-ov-file#platform) project for instructions.
+
+## Getting Started
+
+To get started with the AWS IoT Device SDK for Java v2:
+
+1. Add the SDK to your project - See the [Using SDK](#using-sdk) section for Maven dependency details
+
+2. Choose your connection method - The SDK supports multiple authentication methods including X.509 certificates, AWS credentials, and custom authentication. [MQTT5 User Guide connection section](./documents/MQTT5_Userguide.md#how-to-setup-mqtt5-builder-based-on-desired-connection-method) and [MQTT5 PubSub sample](./samples/Mqtt5/PubSub/README.md) provide more guidance
```

Comment:
Just calling out that the PubSub sample and some of the wording here will need to be changed (including links) once the enhance-samples branch is merged.